### ISSUE TITLE
The first heading on the home page should be `h1`

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#2176]
## Description
We should change the first heading on the homepage to `h1`.

## Additional content
We have audited other pages, and fixed the headings order in #1614
